.circleci/README.asciidoc

```diff
@@ -0,0 +1,70 @@
+== openQA circleci workflow
+
+=== Goal 
+
+Provide a way to run tests with pre-approved list of dependencies both in CI and locally
+
+=== Dependency artefacts
+
+- dependencies.txt list of dependencies to test against.
+- autoinst.sha contains sha of os-autoinst commit for integration testing.
+  When value is empty, the testing will run against latest master
+
+=== Managing and troubleshooting dependencies
+
+dependencies.txt and autoinst.sha are normally managed by dedicated bot, 
+defined in 'nightly' job in config.cnf. The bot manage master branch only;
+and to is possible to add another branch to config.cnf, so daily PR will be 
+submitted with any dependency changes.
+The bot uses build_dependencies.sh script, which can be used manually as well.
+Alternatively just add newly introduced dependencies into dependencies.txt, so CI will 
+run tests with them.
+
+Occasionally it may be a challenge to work with dependencies.txt
+(e.g. package version is not available anymore).
+In such case you can either try to rebuild dependencies.txt using 
+`.circleci/build_dependencies.sh` or just remove all entries and put only
+openQA-devel plus chromedriver into it
+(you can even skip chromedriver if no UI testing is needed).
+Script `.circleci/build_dependencies.sh` can be also modified when major changes are
+performed, e.g. different OS version or packages from forked OBS project, etc.
+
+=== Run tests locally using docker
```

Comment:
I don't see what this section has to do with using "circle CI".

Reply:
The use-case for using Docker locally is to have the same environment than the CI uses. The section describes how to do that.

Reply:
ok but how does it differ from the non-circle CI way to run tests locally using containers?

Reply:
It demonstrates a way to achieve dependencies identical to in CI environment. You don't have to use it normally though.

Reply:
Obviously one needs to *get* the container. That part is specific to what we use in the CI.

Reply:
I love the workflow documentation which outlines how this works and how to reproduce CI locally! I was actually under the impression that this was one of the reasons we were evaluating CircleCI. We didn't have that before.

Reply:
We had that before. (Running UI tests in non-headless mode was broken, though.)